tsconfig-strict.json

```diff
@@ -91,6 +91,7 @@
     "core/templates/services/autoplayed-videos.service.spec.ts",
     "core/templates/services/code-normalizer.service.ts",
     "core/templates/services/code-normalizer.service.spec.ts",
+    "core/templates/services/compute-graph.service.ts",
```

Comment:
Please read the description of #10474 more thoroughly, you forgot to add a spec file here.

Reply:
Actually I read the instructions but there was no spec file of compute-graph.service present

Reply:
Oh, okay, then it is fine.